docs/api/oauth2.md

```diff
@@ -95,6 +96,11 @@ When a top level admin user wants to perform administrative tasks for a
 tenant/organisation, the admin can impersonate the appropriate
 tenant/organisation.
 
+Basically impersonation is allowed top-down; a tenant can impersonate an
+organisation or user but a user can't impersonate any of them.
+Depending on the user's privileges a user may or may not impersonate. (A
+tenant admin user may impersonate the same things a tenant can for example.)
```

Comment:
things=operations

Reply:
Reworded it since I don't think "operations" is a proper fit.